doc/crypto/appendix/history.rst

```diff
@@ -39,6 +39,12 @@ Changes to the API
 
     See :secref:`pake-wpa3-sae`.
 
+Relaxations
+~~~~~~~~~~~
+
+*   Relaxed the permitted-key policy requirements for ECDSA verification, to be consistent with those for ML-DSA and SLH-DSA.
```

Comment:
I think these exceptions should be mentioned in the “Permitted algorithms” section of the chapter on policies. Not necessarily the full list, but at least the general ideas that some algorithms are considered partially equivalent. Maybe just say that it's the case for some signature algorithms, which are only considered equivalent for verification? According to the current text, “A specific algorithm value permits exactly that particular algorithm” (plus NONE and various wildcard cases), which rules out the ML-DSA, SLH-DSA and now ECDSA verification relaxation.

Reply:
Good point - the PQC extension didn't actually modify that text, because it is outside the main spec for now.